tests/offload/tpu_offload_accuracy_test.py

```diff
@@ -90,8 +93,42 @@ def _test_kv_cache_cpu_offloading_accuracy(
         # Waiting for TPUs to be released.
         time.sleep(20)
 
+# This tests scenario where the KV cache size is smaller than the CPU RAM. To ensure a gap the CPU RAM has been set on the higher side while using a smaller prompt.
+# The test does the following
+#       1. generates tokens for the input prompt
+#       2. clears HBM which forces clean up of KV cache from HBM
+#       3. re-calculates tokens for input prompt 
+#       4. verifies length of tokens generated for 1. and 3. when KV cache<CPU RAM
+def test_kv_cache_cpu_offloading_accuracy_smaller_then_cpu_ram(
+    monkeypatch: pytest.MonkeyPatch,
+    sampling_config: SamplingParams,
+    kv_transfer_config: KVTransferConfig,
+):
+    swap_op_types = ["pallas", "jax"]
+    decode_saves = ["0", "1"]
+    skip_precompile = ["0", "1"]
+    for swap_op_type, decode_save, _skip_precompile in itertools.product(
+            swap_op_types, decode_saves, skip_precompile):
+        _test_kv_cache_cpu_offloading_accuracy(
+            monkeypatch,
+            sampling_config,
+            kv_transfer_config,
+            swap_op_type,
+            _skip_precompile,
+            decode_save,
+            # CPU RAM size: 24*256=6144 chunks, cpu_chunk_size: 256
+            "4",
```

Comment:
Please add an explanation of the `cpu_chunk_size`, this represent the number of tokens can fit into a single CPU RAM chuck. The total CPU RAM size = # cpu chucks * cpu_chuck_size = 1024 tokens